platform-service/sample.env

```diff
@@ -1,3 +1,8 @@
+# Flask
+FLASK_APP=src/unstract/platform_service/run.py
+FLASK_RUN_PORT=3001
```

Comment:
@Deepak-Kesavan do we even need these?

Reply:
This will be picked up when we use flask run. Other than that it won't be used. It can be there in OSS repo alone.